src/utilities/convert.ts

```diff
@@ -13,3 +18,40 @@ export function convertObjectValuesToUnit8Array(input: any): Uint8Array {
 export function convertToBase64(input: any): string {
 	return Buffer.from(input).toString("base64");
 }
+
+export function prettifyJSON(
```

Comment:
1. let's rename to `prettifyTransaction`
2. The function should return a new transaction object, rather than overwriting the existing one. So I suggest to start the the function by clonning the transaction, eg using: `Object.assign({}, transaction)`